static/js/util.js

```diff
@@ -293,23 +295,31 @@ export function clean_user_content_links(html) {
         // Ensure that the title displays the real URL.
         let title;
         let legacy_title;
+
         if (url.origin === window.location.origin && url.pathname.startsWith("/user_uploads/")) {
             // We add the word "download" to make clear what will
             // happen when clicking the file.  This is particularly
             // important in the desktop app, where hovering a URL does
             // not display the URL like it does in the web app.
-            title = legacy_title = $t(
-                {defaultMessage: "Download {filename}"},
-                {filename: url.pathname.slice(url.pathname.lastIndexOf("/") + 1)},
-            );
+
+            elt.dataset.id = elt.title;
+            elt.removeAttribute("title");
```

Comment:
Here we have a condition for URL whose pathname starts with `/user_uploads/`, which I think is the URL for the images or content uploaded by users. So, this would only remove the title attribute for the user attachments links if I'm not wrong.